ahref-frontend/src/app/shared/sdk/models/Seat.ts

```diff
@@ -59,6 +61,10 @@ export class Seat implements SeatInterface {
           name: 'column',
           type: 'number'
         },
+        "prva": {
+          name: 'prva',
```

Comment:
Given that 'prva' represents if seat is first class, you should read the project specification which says that not only first class and regular seats exist. There is also a business class, which makes this model inappropriate.